lib/node_helpers.dart

```diff
@@ -12,11 +13,31 @@ Element div({
   AfterCallback afterUpdate,
   AfterCallback afterRemove,
 }) =>
-    _elem('div', attrs, styles, classes, content, events, key, afterInsert,
+    _elem('div', set, attrs, styles, classes, content, events, key, afterInsert,
         afterUpdate, afterRemove);
 
+Element button({
+  /* List<Setter> | Setter */ set,
+  Map<String, String> attrs,
+  Map<String, String> styles,
+  List<String> classes,
+  /* List, Component, Node, BuildFn, ... */ dynamic content,
+  Map<String, EventHandler> events,
+  dynamic key,
+  AfterCallback afterInsert,
+  AfterCallback afterUpdate,
+  AfterCallback afterRemove,
+  EventHandler onClick,
+}) =>
+    _elem('button', set, attrs, styles, classes, content, events, key,
+        afterInsert, afterUpdate, afterRemove)
+      ..onClick(onClick);
+
+Element br() => new Element('br');
+
 Element _elem(
   String tag,
+  List<Setter> set,
```

Comment:
keep this dynamic, as the param is dynamic in the methods

Reply:
Fixed this in https://github.com/tejainece/domino/commit/d44a5a5d185e919aec5ae5614d0b2454bab3f49b